src/amp-story-player/amp-story-player-impl.js

```diff
@@ -716,12 +731,83 @@ export class AmpStoryPlayer {
     new AmpStoryPlayerViewportObserver(this.win_, this.element_, () =>
       this.visibleDeferred_.resolve()
     );
+    if (DESKTOP_PANEL_STORY_PLAYER_EXP_ON) {
+      new this.win_.ResizeObserver((e) => {
```

Comment:
Does this break IE support? Is it polyfilled?

Reply:
It looks like we're deprecating IE11 support #34453.
In that case do you think it's okay to use `ResizeObserver`?

Reply:
I think it's ok if we don't capture the resize events, but does the initial render work correctly for browsers not supporting `ResizeObserver`?